Shared/Samples/Add dynamic entity layer/AddDynamicEntityLayerView.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct AddDynamicEntityLayerView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// A Boolean value indicating whether the settings view should be presented.
+    @State var isShowingSettings = false
+    
+    /// The initial viewpoint for the map.
+    @State var viewpoint = Viewpoint(
+        center: Point(x: -12452361.48631679, y: 4949774.965107439),
```

Comment:
I always love to refer to [xkcd 2170](https://xkcd.com/2170/) when I see sth like this 😆 

![Coordinate Precision](https://imgs.xkcd.com/comics/coordinate_precision.png)

No need to change, but we'll try our best to eliminate these minuscule particles next time when we review a sample design.

Reply:
I love that one....